### ISSUE TITLE
Support GoPro's superview lens correction

### ISSUE THREAD

**Issue by AdrianEddy** [gyroflow/gyroflow#43]
It's 4:3 video squeezed somehow to 16:9, maybe it's anamorphic or something like that. Additional processing is needed to make the lens undistortion work

**Comment by KonradIT** [comment]
There's an open source project that does this: https://github.com/Niek/superview

**Comment by maxbl4** [comment]
+1, would be great to have SuperView checkbox in export settings.
Obviously there are ways to achieve this without gyroflow, but that would be inconvenient and lower quality

**Comment by maxbl4** [comment]
Hm, i guess i misunderstood the original issue. I would simpy like to be able to import 4:3 video, stabilize it and export as superview stretched. Can that be implemented?

**Comment by AdrianEddy** [comment]
Superview is implemented as of ae8ecddf54a9040609cc53de4e5fc6d72b6c3e30
It's not 100% correct, because I still see some minor warping, but I think it's pretty usable.

SuperView is essentially full sensor 4:3 video, non-linearly stretched to 16:9
I took 9 angles of a chessboard with a GoPro in both SuperView and Wide mode (using app so the camera stays in exactly the same spot), then I detected chessboard corners on all 18 images and I tried to figure out an algorithm to make points from SuperView match the ones from Wide shot.
I set up an canvas in html, because we can have live preview in VS Code so the prototyping becomes really quick, I tried to figure out an algorithm and then using Wolfram Alpha I converted the formula to the inverse, so we have the calculation both ways.

I attach the development files, maybe someone wants to pick this up and improve or find out why there's still some warping.

The points are not matching perfectly there, but when I made them aligned better - it was actually producing worse RMS when calibrating the lens, I'm not sure why that is

![image](https://user-images.githubusercontent.com/3849067/165825697-54384054-44be-4600-8d3c-0d8058a2f9e1.png)

[superview-dev.zip](https://github.com/gyroflow/gyroflow/files/8585547/superview-dev.zip)
